app/treatmentcenterMap/ctrl.js

```diff
@@ -42,7 +42,7 @@ function getarrayValue(url) {
         return obj["catg"].join(",");
     }
 }
-function ctrl($scope, $routeParams, service) {
+function ctrl($scope, $routeParams, service, $location) {
```

Comment:
What does this "$location"  here? If you want to use it, then avoid usage of " window.location" in the following code.